mlflow/openai/__init__.py

```diff
@@ -163,6 +164,29 @@ def _get_task_name(task):
         )
 
 
+def _get_api_config() -> Dict[str, Any]:
```

Comment:
I prefere using `dataclass` or `namedtuple` if we know what fields we have.

Reply:
I like that idea!